.github/workflows/build-test.yml

```diff
@@ -1,6 +1,16 @@
 name: build-test
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+    branches:
+      - "main"
```

Comment:
we also need to run them on release branches

Reply:
opened #1188 for that

Reply:
Hmm, I was of the opinion that it doesn't run unless I do that. Let me try removing it.